joker.html

```diff
@@ -8,6 +8,8 @@
 <body>
    Jokes on you Batman!!!
    quick fix on Batman!
+   But this is a fix
    Edited on Github into new branch
+   Nothing Quick about it ...
```

Comment:
I think there should be more here
```suggestion
   Nothing Quick about it ...
```
![3c189d5db77ae082138eb2dcd10d9651](https://user-images.githubusercontent.com/32634730/54023464-b1ca3200-419d-11e9-8d1a-3c100980f8f9.jpg)